package.json

```diff
@@ -55,6 +56,8 @@
     "postcss": "^5.2.0",
     "postcss-discard-duplicates": "^2.0.2",
     "postcss-loader": "^1.1.0",
+    "pouchdb": "^6.1.2",
+    "pouchdb-find": "^0.10.5",
```

Comment:
I know this issue cozy/cozy-client-js#88
But it seems dead, no update since 26 days, almost one full month. No assignee.
May it stay like this forever!

If you see npm complaining about a missing dependency not necessary for your needs, don't you think you must do something else than adding this useless dependency. If we do so, in 6 months, developers will accept that pouchDB is a project's dependency, they will not ask why, they will accept it, as it. I don't think it is something we want. All the more so as it is a big, heavy, large dependency.